extensions/amp-accordion/1.0/accordion.js

```diff
@@ -321,6 +311,75 @@ export function AccordionSection({
     }
   }, [id, toggleExpanded]);
 
+  const context = useMemo(
+    () => ({
+      animate,
+      contentId,
+      expanded,
+      expandHandler,
+    }),
+    [animate, contentId, expanded, expandHandler]
+  );
+
+  return (
+    <Comp {...rest} expanded={expanded}>
+      <SectionContext.Provider value={context}>
+        {children}
+      </SectionContext.Provider>
+    </Comp>
+  );
+}
+
+/**
+ * @param {!AccordionDef.SectionHeaderProps} props
+ * @return {PreactDef.Renderable}
+ */
+export function AccordionSectionHeader({
+  as: Comp = 'header',
+  role = 'button',
+  headerClassName = '',
+  tabIndex = 0,
+  children,
+  ...rest
+}) {
+  const {contentId, expanded, expandHandler} = useContext(SectionContext);
+  const classes = useStyles();
+
+  return (
+    <Comp
+      {...rest}
+      role={role}
+      className={`${headerClassName} ${classes.sectionChild} ${classes.header}`}
+      tabIndex={tabIndex}
+      aria-controls={contentId}
+      onClick={expandHandler}
+      aria-expanded={String(expanded)}
+    >
+      {children}
+    </Comp>
+  );
+}
+
+/**
+ * @param {!AccordionDef.SectionContentProps} props
+ * @return {PreactDef.Renderable}
+ */
+export function AccordionSectionContent({
+  as: Comp = 'div',
+  contentClassName = '',
+  children,
+  ...rest
+}) {
+  const hasMountedRef = useRef(false);
+  const contentRef = useRef(null);
```

Comment:
Can just do `ref` now since there's no ambiguity which ref this is.

Reply:
Updated in latest!